ignite/metrics/vision/object_detection_average_precision_recall.py

```diff
@@ -124,12 +124,16 @@ def box_iou(pred_boxes: torch.Tensor, gt_boxes: torch.Tensor, iscrowd: torch.Boo
         self._area_range = area_range
         self._max_detections_per_image_per_class = max_detections_per_image_per_class
 
-        super(ObjectDetectionAvgPrecisionRecall, self).__init__(
+        # Initialize Metric first
```

Comment:
Why we should do this?
I'm not totally following why we should do this change. If you can please explain it, would be great! Thanks!